inc/switchtec/inject.h

```diff
@@ -0,0 +1,55 @@
+#include <switchtec/switchtec.h>
+#include <stdint.h>
+
+struct switchtec_lnkerr_dllp_in {
```

Comment:
If it's not part of the API, please put it in the c file where it's being referenced.